isis/tests/FunctionalTestsCamdev.cpp

```diff
@@ -96,8 +96,8 @@ TEST_F(LineScannerCube, FunctionalTestCamdevDefault) {
     stddev += oCubeStats->StandardDeviation();
   }
 
-  EXPECT_NEAR(average/oCube.bandCount(), -3.3918738393628582e+306, 0.0000001);
-  EXPECT_NEAR(sum/oCube.bandCount(), 112847425994.19986, 0.0000001);
-  EXPECT_NEAR(stddev/oCube.bandCount(),-3.3918738393628582e+306, 0.0000001);
-  EXPECT_NEAR(validPixels/oCube.bandCount(), 11832.33962264151, 0.0000001);
+  EXPECT_NEAR(average/oCube.bandCount(), 9183553.1942929607, 0.0000001);
+  EXPECT_NEAR(sum/oCube.bandCount(), 112847454091.56306, 0.0000001);
+  EXPECT_NEAR(stddev/oCube.bandCount(), 11.299672545293422, 0.0000001);
+  EXPECT_NEAR(validPixels/oCube.bandCount(), 12064.188679245282, 0.0000001);
```

Comment:
I would like someone to confirm that these changes make sense with the updates made. I am not sure how updating the SolarLongitude changes these outputs.

Reply:
This test generates a band that is the solar longitude at every pixel in the image. Previously this would result in a band of all NULLS, hence the average being astronomically huge, but not it doesn't.

Reply:
See line 70 for the argument that enables this.